CondFormats/Common/test/DropBoxMetadataReader.py

```diff
@@ -0,0 +1,92 @@
+# documentation: https://twiki.cern.ch/twiki/bin/view/CMS/AlCaDBPCL#Drop_box_metadata_management
+
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("myprocess")
+
+process.load("CondCore.CondDB.CondDB_cfi")
+process.load("FWCore.MessageService.MessageLogger_cfi")
+
+process.CondDB.connect = 'sqlite_file:DropBoxMetadata.db' 
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+process.source = cms.Source("EmptySource",
+                            firstRun = cms.untracked.uint32(300000)
+                            )
+
+# process.PoolDBOutputService.DBParameters.messageLevel = 3
+import json
+
+def encodeJsonInString(filename):
+    """This function open the json file and encodes it in a string replacing probelamtic characters"""
+    thefile = open("../data/"+filename)
+    thejson = json.load(thefile)
+    thefile.close()
+    return json.JSONEncoder().encode(thejson).replace('"',"&quot;")
+
+# beamspot by run
+BeamSpotObjectsRcdByRun_prod_str = encodeJsonInString("BeamSpotObjectsRcdByRun_prod.json")
+BeamSpotObjectsRcdByRun_prep_str = encodeJsonInString("BeamSpotObjectsRcdByRun_prep.json")
+
+# beamspot by lumi
+BeamSpotObjectsRcdByLumi_prod_str = encodeJsonInString("BeamSpotObjectsRcdByLumi_prod.json")
+BeamSpotObjectsRcdByLumi_prep_str = encodeJsonInString("BeamSpotObjectsRcdByLumi_prep.json")
+
+# beamspot High Perf by lumi
+BeamSpotObjectsRcdHPByLumi_prod_str = encodeJsonInString("BeamSpotObjectsRcdHPbyLumi_prod.json")
+BeamSpotObjectsRcdHPByLumi_prep_str = encodeJsonInString("BeamSpotObjectsRcdHPbyLumi_prep.json")
+
+# beamspot High Perf by run
+BeamSpotObjectsRcdHPByRun_prod_str = encodeJsonInString("BeamSpotObjectsRcdHPbyRun_prod.json")
+BeamSpotObjectsRcdHPByRun_prep_str = encodeJsonInString("BeamSpotObjectsRcdHPbyRun_prep.json")
```

Comment:
what are all of these above used for?

Reply:
@francescobrivio ?

Reply:
so I dont know what this is doing but I'd like have them removed in a follow-up PR if needed (since this PR is supposed to be technical)

Reply:
@tvami This `test/DropBoxMetadataReader.py` is a new script that you are introducing in this PR and I believe these "beamspot lines" are just a leftover from the previous script that you were modifying (`test/ProduceDropBoxMetadata.py`). 
IIUC they are never used in this script so it should be safe to remove them from here now.

Reply:
Please remove

Reply:
Ahh yes, sorry for the confusion about this, I've remove it now